web/profiles/custom/yalesites_profile/config/sync/gin_lb.settings.yml

```diff
@@ -1,5 +1,8 @@
 _core:
   default_config_hash: 4DJ-ODpqIpFYPDmqzRIVHczK321I99zFqNgPzKwVibs
-toastify_loading: cdn
+toastify_loading: custom
```

Comment:
What's going on here? I didn't see a custom packaged being loaded anywhere.

Reply:
This is just for Gin Layout Builder adding the toastify messages. With it being turned off like this, the status messages are left inline rather than disappearing. This is helpful for things like the reusable notice when editing a reusable block that notice will disappear. Also, it helps with Hector's work with the layout builder lock notification not disappearing and allowing someone to release the lock.